<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1">
    <item>
     <widget class="OpenGLWindow" name="openGLWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Projection Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="perspectiveButton">
           <property name="text">
            <string>Perspective</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="orthographicButton">
           <property name="text">
            <string>Orthographic</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Translation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="zTranslation">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="yTranslation">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="xTranlation">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Rotate</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="xRotate">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-180</number>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="yRotate">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-180</number>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="zRotate">
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="keyboardTracking">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>-180</number>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Scale</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSpinBox" name="scaleSpin">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showVertices">
        <property name="text">
         <string>Show Vertices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Background color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Line color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Screenshot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Load file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="objName">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWindow</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">openglwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
